BayPassTests/Model/Map/RouteTests.swift

```diff
@@ -29,7 +29,7 @@ class RouteTests: XCTestCase {
         
         assert(route.departureTime == depTime)
         assert(route.arrivalTime == arrTime)
-        assert(route.segments == segments)
+//        assert(route.segments == segments)
```

Comment:
Since you removed the Equatable from the RouteSegement this will clearly fail. Commenting out the test case however is not the way to fix this.